recipe/meta.yaml

```diff
@@ -47,7 +47,13 @@ test:
     - pygeoprocessing.routing
     - pygeoprocessing.multiprocessing
   commands:
-    - pytest tests
+    # skipping for now due to change in GDAL exceptions: 
+    # https://github.com/conda-forge/pygeoprocessing-feedstock/pull/35
+    - |
```

Comment:
It looks like this syntax might be breaking on Windows. I'm not sure what you've tried so far but perhaps the YAML folding style could work. 
```yml
commands: >
  pytest
  deselect=tests/test_geoprocessing.py::TestGeoprocessing::test_get_raster_info_error_handling
  deselect=tests/test_geoprocessing.py::TestGeoprocessing::test_get_vector_info_error_handling
  tests
```
I'm pulling this concept from this [SO post](https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines).